warehouse/templates/pages/help.html

```diff
@@ -246,6 +248,11 @@ <h3 id="file-name-reuse">{{ file_name_reuse() }}</h3>
           To avoid this situation, <a href="https://packaging.python.org/guides/using-testpypi/">use Test PyPI to perform and check your upload first</a>, before uploading to <a href="https://pypi.org">pypi.org</a>.
         </p>
 
+        <h3 id="new-classifier">{{ new_classifier() }}</h3>
+        <p>
+          If you would like to request a new Trove classifer please file a bug on our <a href="https://github.com/pypa/warehouse/issues" target="_blank" rel="noopener">issue tracker</a>. Please include the name of the requested classifier and a brief justification of why it is important.
+        </p>
+
```

Comment:
Please add the `{% macro code_of_conduct() %}` block here as it is in other similar answers that are primarily about filing an issue. Thanks.

Reply:
Done in d230e291